common/log/log.go

```diff
@@ -59,8 +59,8 @@ const (
 var (
 	levels = map[int]string{
 		DebugLog: Color(Green, "[DEBUG]"),
-		InfoLog:  Color(Cyan, "[INFO ]"),
-		WarnLog:  Color(Yellow, "[WARN ]"),
+		InfoLog:  Color(Cyan, "[INFO]"),
```

Comment:
this is intended to make the ouput aligned，since `debug`,'error` 'fatal`,and `trace` are 5 letters.

Reply:
Is alignment required?

If so, would maybe adding a `.` or `!` be better than a space?

Reply:
It just looked a little off when I saw it